browser/src/control/Control.ContextToolbar.ts

```diff
@@ -47,7 +47,7 @@ class ContextToolbar {
 	}
 
 	showContextToolbar(): void {
-		if (this.builder.map.isReadOnlyMode()) return;
+		if (this.builder.map.isReadOnlyMode() || !window.mode.isDesktop()) return;
```

Comment:
tablet?

Reply:
I think it should be "non-mobile"

Reply:
to me it didn't feel really nice on tablet though I don't have a real tablet to get the actual feeling... 

cc: @pedropintosilva